context/common_function.go

```diff
@@ -44,6 +44,10 @@ func InTaiList(servedTai models.Tai, taiList []models.Tai) bool {
 	return false
 }
 
+func IsTaiEqual(servedTai models.Tai, targetTai models.Tai) bool {
+	return servedTai.PlmnId == targetTai.PlmnId && servedTai.Tac == targetTai.Tac
```

Comment:
Given that `PlmnId` is a pointer, should not the comparison be as follows?
(of course, you need to make sure that the pointers are not `nil`)
```suggestion
	return servedTai.PlmnId.Mcc == targetTai.PlmnId.Mcc && servedTai.PlmnId.Mnc == targetTai.PlmnId.Mnc && servedTai.Tac == targetTai.Tac
```